ironfish-cli/src/commands/deposit.ts

```diff
@@ -167,12 +170,12 @@ The memo will contain the graffiti "${graffiti}".
 
       const transaction = result.content
       this.log(`
-Depositing ${displayIronAmountWithCurrency(IRON_TO_SEND, true)} from ${
+Depositing ${displayAmount} from ${
         transaction.fromAccountName
       }
 Transaction Hash: ${transaction.hash}
-Transaction fee: ${displayIronAmountWithCurrency(feeInIron, true)}
-
+Transaction fee: ${displayFee}
+New Balance: ${displayNewBalance}
 Find the transaction on https://explorer.ironfish.network/transaction/${
         transaction.hash
```

Comment:
clean up the lint - looks like you may have something off in your tab/space settings

Reply:
@lwisne done. Cleaned it up!